packages/inventory-insights/src/Api.js

```diff
@@ -0,0 +1,25 @@
+import axios from 'axios';
+
+export default {
```

Comment:
no we don't use all of these, but 🤷 doesn't hurt to have em around

Reply:
You should consider using [@redhat-cloud-services/insights-client](https://www.npmjs.com/package/@redhat-cloud-services/insights-client) at some point. No need to call URLs directly.